Makefile

```diff
@@ -88,6 +88,11 @@ repl: ## Python REPL
 outdated: ## List outdated dependancies
 	tox -e py38 -- pip list --outdated
 
+.PHONY: use-docker
+use-docker: ## Use Docker-based local development workflow
```

Comment:
This leaves my `git status` _dirty_. I wonder if that's avoidable?

Reply:
An alternative would be to have both tox and Docker targets in the same Makefile, and switching which targets are executed based on a variable. That would still leave the variable as a potential dirty status candidate though, and the Makefile would be really long.